src/m365/spo/commands/site/site-commsite-enable.spec.ts

```diff
@@ -6,35 +6,34 @@ import { Cli } from '../../../../cli/Cli';
 import { CommandInfo } from '../../../../cli/CommandInfo';
 import { Logger } from '../../../../cli/Logger';
 import Command, { CommandError } from '../../../../Command';
-import config from '../../../../config';
 import request from '../../../../request';
 import { pid } from '../../../../utils/pid';
 import { session } from '../../../../utils/session';
 import { sinonUtil } from '../../../../utils/sinonUtil';
-import { spo } from '../../../../utils/spo';
 import commands from '../../commands';
 const command: Command = require('./site-commsite-enable');
 
 describe(commands.SITE_COMMSITE_ENABLE, () => {
-  let cli: Cli;
   let log: any[];
   let logger: Logger;
   let commandInfo: CommandInfo;
 
+  const defaultPostCallsStub = (): sinon.SinonStub => {
```

Comment:
Since this function is only used once, I don't think it's worth it to centralize it. Let's move it to the test itself.